controllers/cluster_controller_test.go

```diff
@@ -64,6 +64,43 @@ func reloadCluster(cluster *fdbv1beta2.FoundationDBCluster) (int64, error) {
 	return generations.Reconciled, err
 }
 
+func generationsNotMatching(cluster *fdbv1beta2.FoundationDBCluster) func() {
```

Comment:
We can adjust this method to not return a closure but actually perform all the steps.